eo-maven-plugin/src/test/java/org/eolang/maven/rust/ModuleTest.java

```diff
@@ -62,7 +62,7 @@ void transformsCorrectly(@TempDir final Path temp) throws Exception {
                 "use jni::sys::{jint};",
                 "use jni::JNIEnv;",
                 "#[no_mangle]",
-                "pub extern \"system\" fn Java_EOorg_EOeolang_EOrust_simple(_env: JNIEnv, _class: JClass,) -> jint {"
+                "pub extern \"system\" fn Java_EOrust_natives_simple_simple(_env: JNIEnv, _class: JClass,) -> jint {"
```

Comment:
@levBagryansky Just curious why the name was formed like that?

Reply:
@Graur This is how the function must be named depending on the java functon name. For example, if the class `Cat` has native method `say` in package `com.org`, the name will be smth like `Java_com_org_Cat_say`